content/concepts/introduction/protocols/multistream.md

```diff
@@ -1,7 +1,66 @@
 ---
 title: "Multistream Select"
-description: ""
+description: "Multistream Select is used to negotiate and upgrade the protocol of a connection between two peers in libp2p."
 weight: 26
 ---
 
-Coming soon!
+## Overview
+
+[Multistream Select](https://github.com/multiformats/multistream-select) is
+a protocol for friendly protocol multiplexing that allows for the negotiation
```

Comment:
What is unfriendly protocol multiplexing? 🙃

Reply:
I suppose multiplexing that does not graceful handle protocol changes, I removed friendly though.